static/js/jira-select-server.js

```diff
@@ -1,5 +1,8 @@
 $('.select-server').click(function (event) {
   event.preventDefault();
 
-  // TODO: Need to add the logic for navigation
+  AP.navigator.go("addonmodule", {
+    moduleKey: "github-list-server-apps-page",
+    customData: { serverUrl: $(event.target).data("server-name") }
```

Comment:
is it supposed to be `url` or `name?

Reply:
The component that has this attribute is used by both GHE Servers and GH Server apps, for GHE Server, it is serverUrl, but for GH Server apps, it will be app name.
So, `server-name` is an attempt to fit for both cases..

Reply:
Updated the name to `identifier`..